static/images/broken.svg

```diff
@@ -0,0 +1,104 @@
+<svg id="broken" class="slds-illustration" viewBox="0 0 272 146" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
According to https://lightningdesignsystem.com/downloads/ the SLDS images are licensed under https://creativecommons.org/licenses/by-nd/4.0/ so we probably need to link to that.

Reply:
Copyright added:

https://github.com/SFDO-Tooling/MetaCI/pull/411/commits/77cb7d05190fc5ee7f486d015d278197827ea223